### PR TITLE
SIN-20 Singapore returning for 5th year

### PR DESCRIPTION
**Pull request by sergiubodiu**
Announce SIN-20 Singapore returning for 5th year

### CODE REVIEW

File: content/events/2020-singapore/propose.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Type = "event"
+Description = "Propose a talk for devopsdays Singapore 2020"
++++
+  {{< cfp_dates >}}
+
+<hr>
+
+<center>
+
+<a href="https://www.papercall.io/dodsin20" target="_blank" class="btn btn-primary btn-lg">Click here to submit your  proposal!</a>
```
Comment (bridgetkromhout) [inline content/events/2020-singapore/propose.md]: This link does not work yet.

File: content/events/2020-singapore/sponsor.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
++++
+Title = "Sponsor"
+Type = "event"
+Description = "Sponsor devopsdays Singapore 2020"
++++
+
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}] or [download the prospectus](https://drive.google.com/open?id=TBC) right away.
```
Comment (bridgetkromhout) [inline content/events/2020-singapore/sponsor.md]: This prospectus link does not work yet.

File: content/events/2020-singapore/sponsor.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+Singapore is not only one of the fastest growing startup hubs in the world but is also a trusted healthcare, telecom, and financial services nexus in Asia with strong investment in technology from the Government. We aim to attract representation from large enterprises and the startup community alike.
+
+By sponsoring DevOpsDays, you will be exposing your organization’s brand and services to Singapore and Southeast Asia’s top talent in this rapidly growing space.
+
+Sponsoring DevOpsDays Singapore connects your company to some of the greatest technical and business minds in the industry who are influencing decisions in security, monitoring, operations, engineering, quality, and organisational change.
+
+## The Venue
+
+DevOpsDays Singapore 2020 venue TBC shortly
```
Comment (bridgetkromhout) [inline content/events/2020-singapore/sponsor.md]: You cannot announce dates until you also have a venue.

**Comment by bridgetkromhout** [comment]
Hi, @sergiubodiu - if you would like to announce dates, you must also be ready to announce the address of your confirmed venue. If you are still figuring out the venue, then you can list the event without the dates. Then you can add the dates later when you have confirmed your venue.

File: data/events/2020-singapore.yml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Sergiu Bodiu"
+    employer: "Standard Chartered"
+    twitter: "sergiu_bodiu"
+    bio: "Sergiu Bodiu is a Cloud Architect at Standard Chartered. A frequent speaker and program committee member for tech conferences, he leads the devops community at home in Singapore."
+    image: "sergiu-bodiu.jpg"
```
Comment (bridgetkromhout) [inline data/events/2020-singapore.yml]: These will be broken images if not included in `static/events/2020-singapore/organizers/`

File: content/events/2020-singapore/propose.md (hunk 1 above)
Comment (sergiubodiu) [reply content/events/2020-singapore/propose.md]: It went live Today, please verify again

File: content/events/2020-singapore/sponsor.md (hunk 2 above)
Comment (sergiubodiu) [reply content/events/2020-singapore/sponsor.md]: removed the link until we confirm the prospectus.

File: content/events/2020-singapore/sponsor.md (hunk 3 above)
Comment (sergiubodiu) [reply content/events/2020-singapore/sponsor.md]: Removed the dates for now, we have a few venues for our dates, will update once we lock the dates

File: data/events/2020-singapore.yml (hunk 4 above)
Comment (sergiubodiu) [reply data/events/2020-singapore.yml]: thanks for pointing out, I've added the images into the correct folder

**Comment by sergiubodiu** [comment]
Hi @bridgetkromhout made the suggested changes, can you review it again

**Comment by bridgetkromhout** [comment]
Hi, @sergiubodiu - I looked at the preview on https://deploy-preview-8901--devopsdays-web.netlify.com/events/2020-singapore/welcome/ - you can see it there too. You can see that there is a link to a logo which hasn't been uploaded, so it's broken.

On https://deploy-preview-8901--devopsdays-web.netlify.com/events/2020-singapore/sponsor you have "Download Sponsorship Prospectus" without a link - you may not want that?

I noticed that you're listing an email address of "organizers-singapore-2020@devopsdays.org" on https://www.papercall.io/dodsin20 - you will want to use singapore@devopsdays.org. We are not setting up year-specific emails like "organizers-singapore-2020@devopsdays.org" anymore.

**Comment by bridgetkromhout** [comment]
Hi, @sergiubodiu - if you can fix these last few things we could get this merged! Thanks.

**Comment by bridgetkromhout** [comment]
Okay, I see the email address has been updated on papercall. I'm going to merge this in and upload the most recent event logo you used (which you can then change if desired).

**Pull request merged**